src/Reflection/Definition.agda

```diff
@@ -34,6 +38,21 @@ open import Agda.Builtin.Reflection public
            ; data-cons   to constructor′
            ; prim-fun    to primitive′ )
 
+------------------------------------------------------------------------
+-- Showing
+
+show : Definition → String
+show (function cs) = "function " ++ braces (Term.showClauses cs)
+show (data-type pars cs) =
+ "datatype " ++ NatShow.show pars ++ " " ++
+ braces (String.concat (intersperse ", " (map Name.show cs)))
+show (record′ c fs) =
+ "record " ++ Name.show c ++ " " ++
+ braces (String.concat (intersperse ", " (map (Name.show ∘′ Arg.unArg) fs)))
```

Comment:
Please add `intersperse : String → List String → String` to `Data.String.Base`.

Reply:
Done! Also noted that `unlines = intersperse "\n"`